src/LinuxTracing/TracerThread.cpp

```diff
@@ -506,6 +512,27 @@ bool TracerThread::OpenInstrumentedTracepoints(const std::vector<int32_t>& cpus)
   return !tracepoint_event_open_errors;
 }
 
+static std::vector<ThreadName> RetrieveInitialThreadNamesSystemWide(uint64_t initial_timestamp_ns) {
```

Comment:
Why was `RetrieveInitialThreadNamesSystemWideAndNotifyListener` changed to this static function? Couldn't just the logic be changed? This is now a bit inconsistent with `RetrieveInitialTidToPidAssociationSystemWide` and `RetrieveInitialThreadStatesOfTarget`.

Reply:
This was clang-tidy warning I believe telling that this function can be made status, since it does not use anything from the class. I can change others as well. I guess I did not notice because wasn't modifying them.

Reply:
What I meant is that before the function was also notifying the listener, hence using a member of the class. I guess you like it better now, but what I'm saying is that it wasn't really necessary as having a method with side effects instead was consistent with the other two methods I mentioned.

Reply:
yes my personal preference is to do it this way, I think object-file static is more private than the private in the class (because it does not have to be in the header).  And looking at the other functions it seems they can be modified in the same way, they do not have to call the listener - it could be done on the caller side.